paypalmx.php

```diff
@@ -441,7 +441,17 @@ public function hookPayment($params)
 	public function hookDisplayHeader()
 	{
 	  $this->context->controller->addCSS($this->_path.'css/paypal-mx_frontend-overrides.css', 'all');
-	} 
+
+	  // Adds 1.5-specific styles
+		if (version_compare(_PS_VERSION_, '1.6', '<')){
```

Comment:
These *1.5-specific styles* will be inserted on PS 1.4 too, if this not expected, please update your test